lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -74,7 +74,7 @@ <h2 class="hd hd-2 sr" id="details-heading-${course_overview.number}">${_('Cours
           <img src="${course_overview.image_urls['small']}" class="course-image" alt="${_('{course_number} {course_name} Cover Image').format(course_number=course_overview.number, course_name=course_overview.display_name_with_default)}" />
         </a>
       % endif
-      % if settings.FEATURES.get('ENABLE_VERIFIED_CERTIFICATES') and course_verified_certs.get('display_mode') != 'audit':
+      % if display_course_modes_on_dashboard and course_verified_certs.get('display_mode') != 'audit':
```

Comment:
Can/should `course_verified_certs.get('display_mode') != 'audit'` be moved from this template into the criteria for `display_course_modes_on_dashboard`, while we're doing this work?